### ISSUE TITLE
Please add keyboard accellerators/shortcuts to menus

### ISSUE THREAD

**Issue by GoogleCodeExporter** [transmission-remote-gui/transgui#317]
```
Please keyboard enable the menus 'through-out' (well mostly)...like
Example:
**Priority menu shows:
Low
Normal
High 
----
as per 'windows convention, the 1st letter would normally be underlined, and 
pressing 'L', 'N' or 'H' would select the option (after one brings up the 
priority menu)....

Or **right-click on torrent** -- you have 'accelerator' or 'short-cut' keys for 
'Del'/Shift-Del,

but not for (highlighting what I consider to be the best characters)
_S_ tart
S _t_ op
_O_ pen
_P_ riority
Re _A_ nnounce
_V_ erify
P _r_ operties (chose 'r' to be consistent with properties elsewhere in win7)
Setup _C_olumns

Under the 'Tools' Menu:
_O_ptions
_T_ransmission Options
Update _B_locklist
Setup _C_olunms (consistent with right-click)
Update _G_eoIP...

Don't see as much need under the 'Torrent' main menu as they are
duplicate or less used options, but keeping consistency with right-click
if you did,

And for main menu selects
ALT-T for Torrent menu, Alt-O for T_o_ols, and _H_/F1 for Help

you get the idea, I'm sure....if you have any other menus you'd want me to go 
over for suggestions, I'd be happy to follow up...

Basically, it's trying to make the GUI more usable -- like setting priority -- 
it's easy to right click with one hand and select H/N/L w/other, and move down 
a bit, but to have to completely move off column makes fast, _individual_, 
setting impossible (though if all being set to the same, can use group select!  
kudos on that addition!)


```

Original issue reported on code.google.com by `athena.a...@gmail.com` on 25 Sep 2010 at 7:00
